sunpy/map/tests/test_map_factory.py

```diff
@@ -287,3 +280,22 @@ def test_no_2d_hdus(tmpdir):
 
     with pytest.warns(SunpyUserWarning, match='One of the arguments failed to parse'):
         sunpy.map.Map([tmp_fpath, AIA_171_IMAGE], silence_errors=True)
+
+
+@skip_glymur
+def test_map_jp2():
+    jp2_map = sunpy.map.Map(AIA_193_JP2, memmap=False)
+    assert isinstance(jp2_map, sunpy.map.GenericMap)
+    assert jp2_map.data.base is not None
```

Comment:
Could you leave a comment here explaining what `.base` is effectively checking for/why it's `not None` for both memmap cases in this test?

Reply:
Will do.